From: Matteo F. Vescovi <mfv.debian@gmail.com>
Date: Thu, 5 Jan 2012 09:50:48 +0100
Subject: install_in_usr_lib

 * Arch-specific files go to /usr/lib rather than /usr/share
 * Make blender look for systemdir into /usr/lib instead of /usr/share
---
 intern/ghost/intern/GHOST_SystemPathsX11.cpp |    2 +-
 source/creator/CMakeLists.txt                |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

--- blender-2.63.orig/intern/ghost/intern/GHOST_SystemPathsX11.cpp
+++ blender-2.63/intern/ghost/intern/GHOST_SystemPathsX11.cpp
@@ -47,7 +47,7 @@
 #endif
 
 #ifdef PREFIX
-static const char *static_path= PREFIX "/share";
+static const char *static_path= PREFIX "/lib";
 #else
 static const char *static_path= NULL;
 #endif
--- blender-2.63.orig/source/creator/CMakeLists.txt
+++ blender-2.63/source/creator/CMakeLists.txt
@@ -279,8 +279,8 @@ if(WITH_PYTHON)
 	endif()
 
 	install(
-		DIRECTORY ${CMAKE_SOURCE_DIR}/release/scripts
-		DESTINATION ${TARGETDIR_VER}
+		DIRECTORY ${CMAKE_SOURCE_DIR}/release/scripts/
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/blender/scripts/
 		PATTERN ".svn" EXCLUDE
 		PATTERN "__pycache__" EXCLUDE
 		PATTERN "${ADDON_EXCLUDE_CONDITIONAL}" EXCLUDE
